packages/expo-image/src/ExpoImage.web.tsx

```diff
@@ -201,27 +207,49 @@ function getFetchPriorityFromImagePriority(priority: ImageNativeProps['priority'
   return priority && ['low', 'high'].includes(priority) ? priority : 'auto';
 }
 
-export default function ExpoImage({
+function Image({
   source,
-  placeholder,
-  contentFit,
+  events,
   contentPosition,
-  onLoad,
-  transition,
-  onError,
-  responsivePolicy,
-  onLoadEnd,
   priority,
-  ...props
-}: ImageNativeProps) {
-  const { aspectRatio, backgroundColor, transform, borderColor, ...style } = props.style ?? {};
-  const [state, handlers] = useImageState(source);
-  const { placeholder: placeholderStyle, image: imageStyle } = useTransition(transition, state);
-
-  const { containerRef, source: selectedSource } = useSourceSelection(source, responsivePolicy);
+  style,
+}: {
+  source?: ImageSource | null;
+  events?: {
+    onLoad: (((event: React.SyntheticEvent<HTMLImageElement, Event>) => void) | undefined)[];
+    onError: ((({ source }: { source: ImageSource | null }) => void) | undefined)[];
+  };
+  contentPosition?: ImageContentPositionObject;
+  blurhashContentPosition?: ImageContentPositionObject;
+  priority?: string | null;
+  style: React.CSSProperties;
+  blurhashStyle?: React.CSSProperties;
+}) {
+  const objectPosition = getObjectPositionFromContentPositionObject(contentPosition);
+  const uri = source?.uri;
+  return (
+    <img
+      src={uri || undefined}
```

Comment:
```suggestion
  const { uri = undefined } = source;
  return (
    <img
      src={uri}
```

Small simplification.